api/src/test/java/uk/gov/dhsc/htbhf/dwp/model/EligibilityRequestTest.java

```diff
@@ -0,0 +1,70 @@
+package uk.gov.dhsc.htbhf.dwp.model;
+
+import org.junit.jupiter.api.Test;
+import uk.gov.dhsc.htbhf.assertions.AbstractValidationTest;
+
+import java.util.Set;
+import javax.validation.ConstraintViolation;
+
+import static uk.gov.dhsc.htbhf.assertions.ConstraintViolationAssert.assertThat;
+import static uk.gov.dhsc.htbhf.dwp.factory.EligibilityRequestTestDataFactory.*;
+import static uk.gov.dhsc.htbhf.dwp.factory.PersonDTOTestDataFactory.aValidPersonBuilder;
+
+class EligibilityRequestTest extends AbstractValidationTest {
+
+    @Test
+    public void shouldValidateEligibilityRequestSuccessfully() {
+        var request = aValidEligibilityRequest();
```

Comment:
We should only be using the `var` keyword when we are explicitly using `new` otherwise it can be ambiguous, so let's change these to `EligibilityRequest`.